templates/sso74-image-stream.json

```diff
@@ -13,165 +13,53 @@
             "kind": "ImageStream",
             "apiVersion": "v1",
             "metadata": {
-                "name": "redhat-sso-cd-openshift",
+                "name": "redhat-sso74-openshift",
                 "annotations": {
-                    "description": "Red Hat Single Sign-On Continuous Delivery",
-                    "openshift.io/display-name": "Red Hat Single Sign-On Continuous Delivery",
+                    "description": "Red Hat Single Sign-On 7.4 on OpenJDK",
+                    "openshift.io/display-name": "Red Hat Single Sign-On 7.4 on OpenJDK",
                     "openshift.io/provider-display-name": "Red Hat, Inc.",
-                    "version": "6"
+                    "version": "7.4.0.GA"
                 }
             },
             "labels": {
-                "rhsso": "6"
+                "rhsso": "7.4.0.GA"
             },
             "spec": {
                 "tags": [
                     {
                         "name": "latest",
                         "annotations": {
-                            "description": "Latest build of Red Hat Single Sign-On Continuous Delivery Tech Preview image",
+                            "description": "Latest build of Red Hat Single Sign-On 7.4 OpenJDK image",
```

Comment:
We should be consistent IMHO. If you want / need use `7.4 on OpenJDK` everywhere (in description, display name etc. fields)